rmf_demo_panel/README.md

```diff
@@ -0,0 +1,52 @@
+# RMF Demo Panel
+
+This will be a replacement of rviz's [rmf_panel](https://github.com/osrf/rmf_schedule_visualizer)
+
+## Installation
+Dependencies
+ - rmf_core: [`develop/dispatcher_demo`](https://github.com/osrf/rmf_demos/tree/develop/dispatcher_demo) branch
+
+Setup `rmf_demo_panel`
+```bash
```

Comment:
Does the user require `nodejs` to install the panel? If so let's add instructions to install the same.